src/serac/numerics/equation_solver.hpp

```diff
@@ -192,6 +192,62 @@ class SuperLUSolver : public mfem::Solver {
   mfem::SuperLUSolver superlu_solver_;
 };
 
+#ifdef MFEM_USE_STRUMPACK
+/**
+ * @brief A wrapper class for using the MFEM Strumpack solver with a HypreParMatrix
+ */
+class StrumpackSolver : public mfem::Solver {
+public:
+  /**
+   * @brief Constructs a wrapper over an mfem::STRUMPACKSolver
+   * @param[in] comm The MPI communicator used by the vectors and matrices in the solve
+   * @param[in] print_level The verbosity level for the mfem::STRUMPACKSolver
+   */
+  StrumpackSolver(int print_level, MPI_Comm comm) : strumpack_solver_(0, nullptr, comm)
+  {
+    strumpack_solver_.SetKrylovSolver(strumpack::KrylovSolver::DIRECT);
+    strumpack_solver_.SetReorderingStrategy(strumpack::ReorderingStrategy::METIS);
+    strumpack_solver_.DisableMatching();
+
+    if (print_level == 1) {
+      strumpack_solver_.SetPrintFactorStatistics(true);
+      strumpack_solver_.SetPrintSolveStatistics(true);
+    }
+  }
+
+  /**
+   * @brief Factor and solve the linear system y = Op^{-1} x using Strumpack
+   *
+   * @param x The input RHS vector
+   * @param y The output solution vector
+   */
+  void Mult(const mfem::Vector& x, mfem::Vector& y) const;
```

Comment:
Should these be called `input` and `output`?

Reply:
Yes - this is an mfem-ism I picked up.